ddtrace/internal/telemetry/writer.py

```diff
@@ -456,6 +456,7 @@ def _app_started_event(self):
                 ("ddtrace_bootstrapped", config._ddtrace_bootstrapped, "unknown"),
                 ("ddtrace_auto_used", "ddtrace.auto" in sys.modules, "unknown"),
                 ("otel_enabled", config._otel_enabled, "unknown"),
+                ("runtimemetrics_enabled", asbool(os.getenv("DD_RUNTIME_METRICS_ENABLED", default=False)), "unknown"),
```

Comment:
Should we use the constant `TELEMETRY_RUNTIMEMETRICS_ENABLED` constant here? Can define this constant in ddtrace/internal/telemetry/constants.py